youtube/generic.md

```diff
@@ -41,15 +41,11 @@ Allan Regush https://github.com/AllanRegush
 
 ## Service I Use and Recommend
 
-Some of these might be affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount too
```

Comment:
This sounds like a disclaimer - are we sure we can leave this out?

Reply:
I think most people write something above their affiliate links, to avoid any confusion